coala_quickstart/green_mode/green_mode_core.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+from coala_quickstart.green_mode.green_mode import (
+    initialize_project_data, dump_to_file, get_contents,
+    run_quickstartbear, bear_test_fun)
+from coala_quickstart.green_mode.filename_operations import (
+    check_filename_prefix_postfix)
+
+
+def green_mode(project_dir, ignore_globs, bears, bear_settings_obj,
```

Comment:
add type annotations.  e.g. I need to know whether `project_dir` is a Path or a str, in order to accurately review this without trying to find all invocations to see how it is being used.